### PR TITLE
Fix branch calculation for re-deploy

### PR DESCRIPTION
**Pull request by valituguran**
Fix branch calculation for re-deploy.

### CODE REVIEW

File: src/org/ods/services/BitbucketService.groovy [hunk 1]
```diff
@@ -367,14 +370,56 @@ class BitbucketService {
...
+                    logger.warn("Could not get Bitbucket repo '${repo}' default branch due to: ${err}")
                 }
             }
         }
-        logger.debugClocked("defaultbranch-${project}-${project}-${repo}")
+        logger.debugClocked("defaultbranch-${repo}")
         return displayId
     }
 
+    Map findRepoBranchesStartingWith(String repo, String filterText) {
```
Comment (hrcornejo) [inline src/org/ods/services/BitbucketService.groovy:381]: Rename to findRepoBranches

File: src/org/ods/component/ScanWithAquaStage.groovy [hunk 2]
```diff
@@ -428,6 +429,16 @@ class ScanWithAquaStage extends Stage {
     private String computeScannedBranch() {
         def scannedBranch = context.getGitBranch()
         if (scannedBranch.toLowerCase().startsWith("release/")) { // We scanned the default integration branch
+            Map branchesResponse = bitbucket.findRepoBranchesStartingWith(context.getRepoName(), scannedBranch)
```
Comment (hrcornejo) [inline src/org/ods/component/ScanWithAquaStage.groovy:432]: Add a comment to indicate that this is to check if the branch has been created in bitbucket.

**Review by hrcornejo — CHANGES_REQUESTED** [review]

File: src/org/ods/services/BitbucketService.groovy (hunk 1 above)
Comment (valituguran) [reply src/org/ods/services/BitbucketService.groovy]: Done.

File: src/org/ods/component/ScanWithAquaStage.groovy (hunk 2 above)
Comment (valituguran) [reply src/org/ods/component/ScanWithAquaStage.groovy]: Done.

**Review by hrcornejo — APPROVED** [review]

**Pull request merged**